pkg/sql/internal.go

```diff
@@ -385,6 +386,9 @@ func (ie *internalExecutorImpl) execInternal(
 	stmt string,
 	qargs ...interface{},
 ) (retRes result, retErr error) {
+	if sargs != nil && sargs.ApplicationName == "" {
+		sargs.ApplicationName = fmt.Sprintf("internal-%s", opName)
```

Comment:
I'm not superplussed about this `Sprintf`.

The `opName` argument is a constant string in all the callers. It's silly (and inefficient, and slow, and expensive) to add the "internal" prefix *at run time* given that the strings are constant.

Instead, change the call points to add the "internal" prefix to the `opName` argument there, in the string constant.